experiment/src/prepare_data.py

```diff
@@ -7,8 +7,8 @@ def execute_scanner(dataset_location: str, result_location_str, j, use_ml=False)
     """Execute CredSweeper as a separate process to make sure no global states is shared with training script"""
     dir_path = os.path.dirname(os.path.realpath(__file__)) + "/.."
     command = f"{sys.executable} -m credsweeper --path {dataset_location}/data --save-json {result_location_str} -j {j}"
-    if use_ml:
-        command += " --ml_validation"
+    if not use_ml:
+        command += " --ml_threshold skip"
```

Comment:
IMHO, 'skip' is extra. 0 is enough. Or empty argument.

Reply:
Okay. I removed `skip` attribute from `ThresholdPreset` with [4960952](https://github.com/Samsung/CredSweeper/pull/146/commits/496095207723d5e087385e74cdc890bba8c5fefc).